index.html

```diff
@@ -109,7 +109,39 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ________ Contributor's card END ________  -->
-        <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+
+        <!-- ________ Contributor's card START ________  -->
+                <div class="card">
+                  <p class="name">Paulo Eduardo</p>
+                  <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/EduMendesBRA" target="_blank">@EduMendesBRA</a>
+                  </p>
+                  <p class="about">
+                   I`m a cool brasilian guy.
+                  </p>
+                  <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                      <li>
+                        <a href="https://comunidadesebrae.com.br/gamification/t" target="_blank" title="Gamefication SEBRAE">
+                          Gamefication SEBRAE
+                        </a>
+                      </li>
+                      <li>
+                        <a href="https://sebraeinteligenciasetorial.com.br/produtos/relatorios-de-inteligencia/gamificacao-como-ferramenta-motivacional-nas-empresas/5f2821a3300d5b1800c4fc60" target="_blank" title="GAMIFICAÇÃO COMO FERRAMENTA MOTIVACIONAL NAS EMPRESAS">
+                          GAMIFICAÇÃO COMO FERRAMENTA MOTIVACIONAL NAS EMPRESAS
+                        </a>
+                      </li>
+                      <li>
+                        <a href="https://yukaichou.com/gamification-examples/octalysis-complete-gamification-framework/" target="_blank" title="Gamification & Behavioral Design">
+                          Gamification & Behavioral Design
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+          <!-- ________ Contributor's card END ________  -->
```

Comment:
Kindly take your entire card and paste it below the line that says "Paste Your Card Directly Below this Line"